Apps/fastlane/Fastfile

```diff
@@ -94,13 +79,18 @@ platform :ios do
       
       firebase_app_distribution(
         service_credentials_file: service_credentials_file_path,
-        groups: test_groups,
-        release_notes: build_notes
+        groups: get_build_test_groups(),
+        release_notes: get_build_notes()
       )
     end 
-  end
-
-  lane :update_ios_app_versions do |arguments| 
+  end  
+  
+  # Given app and build version, update the Xcode project. Usually you call this before building the app. 
+  # There are multiple places to modify the version numbers. This function's job is guarantee when function is done, the version numbers are updated.
+  #
+  # Usage:
+  # `fastlane ios update_app_version build_number:123 app_version:1.2.3`
+  lane :update_app_version do |arguments| 
```

Comment:
Added comments to this function and also renamed it. 

The old name felt verbose: `ios update_ios_app_versions`. 
New name feels better: `ios update_app_version`.